cmd/down.go

```diff
@@ -33,16 +32,9 @@ func down(cmd *cobra.Command, args []string) {
 		log.Fatal("--user is required for the up command")
```

Comment:
The `up` and `down` commands don't work since the root command is still requiring the [`--user` flag](https://github.com/turnerlabs/harbor-compose/blob/master/cmd/root.go#L52) and [User global variable](https://github.com/turnerlabs/harbor-compose/blob/master/cmd/root.go#L33).  We should remove these.